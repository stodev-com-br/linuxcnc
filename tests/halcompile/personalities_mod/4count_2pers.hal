# personalities are used modulo the number of
# halcompiled personalities

# 4 count, 2 personalities
loadrt logic_test    count=4 personality=0x102,0x204
loadrt lincurve_test count=4 personality=2,3
loadrt bitslice_test count=4 personality=2,3

show pin
show parameter
