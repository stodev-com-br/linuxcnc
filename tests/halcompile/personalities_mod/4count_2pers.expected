bitslice_test u32   IN 0 bitslice-test.0.in
bitslice_test bit   OUT FALSE bitslice-test.0.out-00
bitslice_test bit   OUT FALSE bitslice-test.0.out-01
bitslice_test s32   OUT 0 bitslice-test.0.time
bitslice_test u32   IN 0 bitslice-test.1.in
bitslice_test bit   OUT FALSE bitslice-test.1.out-00
bitslice_test bit   OUT FALSE bitslice-test.1.out-01
bitslice_test bit   OUT FALSE bitslice-test.1.out-02
bitslice_test s32   OUT 0 bitslice-test.1.time
bitslice_test u32   IN 0 bitslice-test.2.in
bitslice_test bit   OUT FALSE bitslice-test.2.out-00
bitslice_test bit   OUT FALSE bitslice-test.2.out-01
bitslice_test s32   OUT 0 bitslice-test.2.time
bitslice_test u32   IN 0 bitslice-test.3.in
bitslice_test bit   OUT FALSE bitslice-test.3.out-00
bitslice_test bit   OUT FALSE bitslice-test.3.out-01
bitslice_test bit   OUT FALSE bitslice-test.3.out-02
bitslice_test s32   OUT 0 bitslice-test.3.time
lincurve_test float IN 0 lincurve-test.0.in
lincurve_test float OUT 0 lincurve-test.0.out
lincurve_test float I/O 0 lincurve-test.0.out-io
lincurve_test s32   OUT 0 lincurve-test.0.time
lincurve_test float IN 0 lincurve-test.1.in
lincurve_test float OUT 0 lincurve-test.1.out
lincurve_test float I/O 0 lincurve-test.1.out-io
lincurve_test s32   OUT 0 lincurve-test.1.time
lincurve_test float IN 0 lincurve-test.2.in
lincurve_test float OUT 0 lincurve-test.2.out
lincurve_test float I/O 0 lincurve-test.2.out-io
lincurve_test s32   OUT 0 lincurve-test.2.time
lincurve_test float IN 0 lincurve-test.3.in
lincurve_test float OUT 0 lincurve-test.3.out
lincurve_test float I/O 0 lincurve-test.3.out-io
lincurve_test s32   OUT 0 lincurve-test.3.time
logic_test bit   OUT FALSE logic-test.0.and
logic_test bit   IN FALSE logic-test.0.in-00
logic_test bit   IN FALSE logic-test.0.in-01
logic_test s32   OUT 0 logic-test.0.time
logic_test bit   IN FALSE logic-test.1.in-00
logic_test bit   IN FALSE logic-test.1.in-01
logic_test bit   IN FALSE logic-test.1.in-02
logic_test bit   IN FALSE logic-test.1.in-03
logic_test bit   OUT FALSE logic-test.1.or
logic_test s32   OUT 0 logic-test.1.time
logic_test bit   OUT FALSE logic-test.2.and
logic_test bit   IN FALSE logic-test.2.in-00
logic_test bit   IN FALSE logic-test.2.in-01
logic_test s32   OUT 0 logic-test.2.time
logic_test bit   IN FALSE logic-test.3.in-00
logic_test bit   IN FALSE logic-test.3.in-01
logic_test bit   IN FALSE logic-test.3.in-02
logic_test bit   IN FALSE logic-test.3.in-03
logic_test bit   OUT FALSE logic-test.3.or
logic_test s32   OUT 0 logic-test.3.time

bitslice_test s32   RW 0 bitslice-test.0.tmax
bitslice_test bit   RO FALSE bitslice-test.0.tmax-increased
bitslice_test s32   RW 0 bitslice-test.1.tmax
bitslice_test bit   RO FALSE bitslice-test.1.tmax-increased
bitslice_test s32   RW 0 bitslice-test.2.tmax
bitslice_test bit   RO FALSE bitslice-test.2.tmax-increased
bitslice_test s32   RW 0 bitslice-test.3.tmax
bitslice_test bit   RO FALSE bitslice-test.3.tmax-increased
lincurve_test s32   RW 0 lincurve-test.0.tmax
lincurve_test bit   RO FALSE lincurve-test.0.tmax-increased
lincurve_test float RW 0 lincurve-test.0.x-val-00
lincurve_test float RW 0 lincurve-test.0.x-val-01
lincurve_test float RW 0 lincurve-test.0.y-val-00
lincurve_test float RW 0 lincurve-test.0.y-val-01
lincurve_test s32   RW 0 lincurve-test.1.tmax
lincurve_test bit   RO FALSE lincurve-test.1.tmax-increased
lincurve_test float RW 0 lincurve-test.1.x-val-00
lincurve_test float RW 0 lincurve-test.1.x-val-01
lincurve_test float RW 0 lincurve-test.1.x-val-02
lincurve_test float RW 0 lincurve-test.1.y-val-00
lincurve_test float RW 0 lincurve-test.1.y-val-01
lincurve_test float RW 0 lincurve-test.1.y-val-02
lincurve_test s32   RW 0 lincurve-test.2.tmax
lincurve_test bit   RO FALSE lincurve-test.2.tmax-increased
lincurve_test float RW 0 lincurve-test.2.x-val-00
lincurve_test float RW 0 lincurve-test.2.x-val-01
lincurve_test float RW 0 lincurve-test.2.y-val-00
lincurve_test float RW 0 lincurve-test.2.y-val-01
lincurve_test s32   RW 0 lincurve-test.3.tmax
lincurve_test bit   RO FALSE lincurve-test.3.tmax-increased
lincurve_test float RW 0 lincurve-test.3.x-val-00
lincurve_test float RW 0 lincurve-test.3.x-val-01
lincurve_test float RW 0 lincurve-test.3.x-val-02
lincurve_test float RW 0 lincurve-test.3.y-val-00
lincurve_test float RW 0 lincurve-test.3.y-val-01
lincurve_test float RW 0 lincurve-test.3.y-val-02
logic_test s32   RW 0 logic-test.0.tmax
logic_test bit   RO FALSE logic-test.0.tmax-increased
logic_test s32   RW 0 logic-test.1.tmax
logic_test bit   RO FALSE logic-test.1.tmax-increased
logic_test s32   RW 0 logic-test.2.tmax
logic_test bit   RO FALSE logic-test.2.tmax-increased
logic_test s32   RW 0 logic-test.3.tmax
logic_test bit   RO FALSE logic-test.3.tmax-increased

